src/common/utils.py

```diff
@@ -306,7 +260,7 @@ def _get_instance_info(region, proxy_config, instance_type, additional_instance_
 
 def get_instance_properties(region, proxy_config, instance_type, additional_instance_types_data=None):
```

Comment:
we could also remove this logic since not used in slurm plugin if I recall correctly

Reply:
Done, I removed also other unused utility functions.